credentials/alts/internal/testutil/testutil.go

```diff
@@ -22,11 +22,14 @@ package testutil
 import (
 	"bytes"
 	"encoding/binary"
+	"fmt"
 	"io"
 	"net"
 	"sync"
 
+	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/credentials/alts/internal/conn"
+	altspb "google.golang.org/grpc/credentials/alts/internal/proto/grpc_gcp"
```

Comment:
Same comment about splitting imports for messages and services.

Reply:
Done.